ui/src/path.ts

```diff
@@ -45,7 +45,7 @@ export const projectSource = (
   if (revision && path) {
     return `/projects/${id}/source/${revision}/${objectType}/${
       objectType === ObjectType.Tree ? `${path}/` : path
-    }`;
+      }`;
```

Comment:
Is this a mistake reformatting?

Reply:
Yeah it's automatically putting it there on save 🤷‍♂️

Reply:
What's your `prettier --version`?